relay-redis/src/real.rs

```diff
@@ -22,6 +26,7 @@ pub enum RedisError {
 
 enum ConnectionInner<'a> {
     Cluster(&'a mut redis::cluster::ClusterConnection),
+    MultiWrite(Box<ConnectionInner<'a>>, Vec<ConnectionInner<'a>>),
```

Comment:
Could name the fields here:

```suggestion
    MultiWrite { primary: Box<ConnectionInner<'a>>, secondary: Vec<ConnectionInner<'a>> },
```